draft-packages/tag/docs/Tag.stories.tsx

```diff
@@ -36,10 +36,10 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
   <>
     <StoryWrapper isReversed={isReversed}>
       <StoryWrapper.RowHeader
-        gridColumns={10}
-        headings={["Default", "Live", "Draft"]}
+        gridColumns={8}
+        headings={["Default", "Live", "Draft", "Action", "Closed"]}
```

Comment:
@jwmccowan Cleared it with the team and since it's already a breaking change would be great if we could update the vairant from `statusAction` to `statusActionRequired` to make it more explicit with its status. Also updating this heading to "Action Required" above :)

Reply:
Thanks for reverting this